src/components/forms/event-form.js

```diff
@@ -437,6 +439,17 @@ class EventForm extends React.Component {
         });
     }
 
+    getPopupScores(score_id) {
+        const {entity} = this.state;
+        let res = ''
+        const rating_type = entity?.selection_plan?.track_chair_rating_types.find((st) => st.id === parseInt(score_id));        
```

Comment:
what happen if the entity.selection_plan is null or undefined ?
rating type is null or undef too and i do think will break on line ln#446